graylog2-web-interface/src/views/hooks/SyncWithQueryParameters.js

```diff
@@ -31,9 +31,9 @@ const extractTimerangeParams = (timerange: TimeRange) => {
 };
 
 export const syncWithQueryParameters = (query: string, action: (string) => mixed = history.push) => {
-  const { view } = ViewStore.getInitialState() || {};
+  const { view, dirty } = ViewStore.getInitialState() || {};
 
-  if (view && view.type === View.Type.Search) {
+  if (view && view.type === View.Type.Search && dirty) {
```

Comment:
As noted in a DM, optional chaining was not available when this was initially written, but could be simplified a little bit with 
```suggestion
  if (view?.type === View.Type.Search && dirty) {
```